cron_jobs/cronUniversityPayments.ts

```diff
@@ -29,14 +29,18 @@ const sendMailToUniversities = async () => {
       );
       console.log(`Summary sent for ${university.name} - ${logs.hash(emailsTo)}`);
     }
+
+    return Promise.resolve();
   });
+
+  await Promise.all(sendMails);
```

Comment:
Since you are "awaiting'" it in `sendSummaryToUniversities`, shouldn't you simply `return Promise.all(sendMails);` here?